packages/manager/apps/dedicated/client/app/dedicated/server/ftp-backup/dedicated-server-ftp-backup.html

```diff
@@ -8,18 +8,18 @@
             <div data-ovh-alert="server_tab_ftpbackup_alert"></div>
 
             <p
-                data-ng-if="!featureAvailable"
+                data-ng-if="!featureAvailable || isNotAvailable"
                 data-translate="server_configuration_ftpbackup_not_available"
             ></p>
 
             <div
                 class="alert alert-info"
                 role="alert"
-                data-ng-if="!ftpBackup.model && featureAvailable"
+                data-ng-if="isNotActivatedButActivable && featureAvailable"
```

Comment:
**suggestion**: `isNotActivatedButActivable` variable can be renamed to `canBeActivated`